DESCRIPTION

```diff
@@ -18,5 +18,6 @@ Imports:
     dplyr,
     googlesheets4,
     magrittr,
+    pxRRead,
     tibble,
     tidyr
```

Comment:
```suggestion
 Imports: 
    assertthat,
    BFS,
    dplyr,
    googlesheets4,
    magrittr,
    tibble,
    tidyr
Remotes:  
    SDSC-ORD/pxRRead
```

adding pxRRread package as Github dependency. Otherwise package won't load with `devtools::load_all()` if package is not already installed locally.